src/main/java/com/google/devtools/build/lib/exec/ExecutionOptions.java

```diff
@@ -318,11 +317,27 @@ public boolean shouldMaterializeParamFiles() {
               + "(eg. HOST_RAM*.5 to use half the available RAM)."
               + "By default, (\"HOST_RAM*.67\"), Bazel will query system configuration to estimate "
               + "amount of RAM available for the locally executed build actions and will use 67% "
-              + "of available RAM. "
-              + "Note: This is a no-op if --local_resources is set.",
+              + "of available RAM.",
       converter = RamResourceConverter.class)
   public float localRamResources;
 
+  @Option(
+      name = "local_extra_resources",
+      defaultValue = "null",
+      documentationCategory = OptionDocumentationCategory.UNCATEGORIZED,
+      effectTags = {OptionEffectTag.UNKNOWN},
+      allowMultiple = true,
+      help =
+          "Set the number of extra resources available to Bazel. "
+            + "Takes in a string-float pair. Can be used multiple times to specify multiple "
+            + "types of extra resources. Bazel will limit concurrently running test actions "
```

Comment:
You mentioned test actions here. If this param only for tests, then it's better to add `test` in its name. E.g. `local_test_extra_resources`. This is also applies to name of this field in classes

Reply:
Hmm, I guess the implementation should not be restricted to tests, although I have not verified that it works for non-test actions as well. Let me investigate this a bit more. Would be nice to match the behavior and naming of `--local_cpu_resources` and `--local_ram_resources` which also apply to both test and build actions?

Reply:
Yeah, this resources are implied not only for test actions. Could you please then remove `test` from description or use text about tests as an example.